package.json

```diff
@@ -15,15 +15,16 @@
     "marked": "^0.3.12",
     "minimist": "^1.2.0",
     "sander": "^0.6.0",
+    "ts-node": "^8.3.0",
```

Comment:
ts-node transpiles `.ts` files on the fly (used for cli.js because it was easy). Compiled files are cached, so they don't re-compile on second run (plus the performance hit is not noticeable because the lib isn't big enough for it to matter).